### ISSUE TITLE
Older consoles: "+1 MU". Consoles from MidnightSun: "+1MU"

### ISSUE THREAD

**Issue by Kawau** [Null-Signal-Games/netrunnerdb#730]
When searching for consoles by their text, you can't find for ex. Marrow by typing `x:"+1 MU"`:
![image](https://user-images.githubusercontent.com/10105234/217856883-d91d53fa-1bb2-4f43-bacc-b6fa35b52045.png)

Instead, you have to search by `x:"+1MU"` (without the space between the number and "MU"):
![image](https://user-images.githubusercontent.com/10105234/217857340-657af201-3868-40d9-b2ec-cd7cd9afd376.png)
 
So basically, Marrow, Virtuoso and Endurance have different text than older consoles :)

**Comment by plural** [comment]
this is actually an issue for our JSON repo, but thank you for reporting it!
